documentation/modules/appendix_crds.adoc

```diff
@@ -781,10 +783,67 @@ It must have the value `password` for the type `KafkaJmxAuthenticationPassword`.
+
+
+[options="header"]
+|====
+|Property           |Description
+|outputType  1.2+<.<|Template for setting the format of the data that will be pushed.
+|string
+|host        1.2+<.<|The ip of the remote host that the data will be pushed to.
+|string
+|port        1.2+<.<|The port of the remote host that the data will be pushed to.
+|integer
+|flushDelay  1.2+<.<|How many seconds the JmxTransSpec will wait before pushing a new set of data out.
+|integer
+|typeNames   1.2+<.<|Template for being able to filter out exactly what data you want when a wildcard query is sent. More information can be found here: https://github.com/jmxtrans/jmxtrans/wiki/Queries.
+|string array
+|name        1.2+<.<|Template for setting the name of the output definition which will be used to know where to send the data of the queries to.
+|string
+|====
+
+[id='type-JmxTransQueryTemplate-{context}']
+### `JmxTransQueryTemplate` schema reference
+
+Used in: xref:type-JmxTransSpec-{context}[`JmxTransSpec`]
+
+
+[options="header"]
+|====
+|Property            |Description
+|targetMBean  1.2+<.<|If using a wildcard, the MBean(s) hat the data will be gathered from.
+|string
+|attributes   1.2+<.<|attributes determine which attributes of the targeted MBean will be read.
+|string array
+|outputs      1.2+<.<|outputs is a list of names of the Output Definitions which configures where the data will be pushed to and what data format it will be in.
+|string array
+|====
+
 [id='type-ResourceRequirements-{context}']
 ### `ResourceRequirements` schema reference
 
-Used in: xref:type-EntityTopicOperatorSpec-{context}[`EntityTopicOperatorSpec`], xref:type-EntityUserOperatorSpec-{context}[`EntityUserOperatorSpec`], xref:type-KafkaBridgeSpec-{context}[`KafkaBridgeSpec`], xref:type-KafkaClusterSpec-{context}[`KafkaClusterSpec`], xref:type-KafkaConnectS2ISpec-{context}[`KafkaConnectS2ISpec`], xref:type-KafkaConnectSpec-{context}[`KafkaConnectSpec`], xref:type-KafkaExporterSpec-{context}[`KafkaExporterSpec`], xref:type-KafkaMirrorMakerSpec-{context}[`KafkaMirrorMakerSpec`], xref:type-TlsSidecar-{context}[`TlsSidecar`], xref:type-TopicOperatorSpec-{context}[`TopicOperatorSpec`], xref:type-ZookeeperClusterSpec-{context}[`ZookeeperClusterSpec`]
+Used in: xref:type-EntityTopicOperatorSpec-{context}[`EntityTopicOperatorSpec`], xref:type-EntityUserOperatorSpec-{context}[`EntityUserOperatorSpec`], xref:type-JmxTransSpec-{context}[`JmxTransSpec`], xref:type-KafkaBridgeSpec-{context}[`KafkaBridgeSpec`], xref:type-KafkaClusterSpec-{context}[`KafkaClusterSpec`], xref:type-KafkaConnectS2ISpec-{context}[`KafkaConnectS2ISpec`], xref:type-KafkaConnectSpec-{context}[`KafkaConnectSpec`], xref:type-KafkaExporterSpec-{context}[`KafkaExporterSpec`], xref:type-KafkaMirrorMakerSpec-{context}[`KafkaMirrorMakerSpec`], xref:type-TlsSidecar-{context}[`TlsSidecar`], xref:type-TopicOperatorSpec-{context}[`TopicOperatorSpec`], xref:type-ZookeeperClusterSpec-{context}[`ZookeeperClusterSpec`]
 
 
 [options="header"]
```

Comment:
I think we're missing the descriptions here.

![image](https://user-images.githubusercontent.com/47596553/71826697-666f0180-3096-11ea-9b7f-2fde32d85ae5.png)

Reply:
I'm not sure where the descriptions are in the code? This table is for `ResourceRequirements` schema reference

Reply:
There doesn't seem to be any description of those in master branch either? https://github.com/strimzi/strimzi-kafka-operator/blob/master/documentation/modules/appendix_crds.adoc